src/client/services/gitHubService.ts

```diff
@@ -0,0 +1,26 @@
+import GitHub from 'github-api';
+
+export interface IGithubService {
+  /**
+   * Returns the gist of the latest commit for the given owner/repo.
+   */
+  getRepoLastCommitGist(owner: string, repo: string): Promise<{ message: string }>;
+}
+
+export class GitHubService implements IGithubService {
```

Comment:
Rename this file to `GitHubService.ts`, same as the class name.

Reply:
Done